src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -276,9 +278,17 @@ extended_prefix_map:
     prefix_synonyms: []
     uri_prefix: http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/KEGG/
     uri_prefix_synonyms: []
-  - prefix: ICD11
```

Comment:
you dont have to delete this, as ICD11 stands for "linearisation of ICD11" which we still need for ORDO and DOID xref processing.

Reply:
NP. I thought deletion of this was approved. I'll restore it.